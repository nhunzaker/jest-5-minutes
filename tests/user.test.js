import User from '../src/user'

test('calculates the full name', function () {
  let user = new User({
    first: 'Billy',
    last: 'Booster'
  })

  expect(user.fullname).toEqual('Billy Booster')
})

test('serializes to a JSON structor', function () {
  let user = new User({
    first: 'Billy',
    last: 'Booster'
  })

  expect(user).toMatchSnapshot()
})
