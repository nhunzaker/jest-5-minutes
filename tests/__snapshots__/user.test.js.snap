// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serializes to a JSON structor 1`] = `
Object {
  "first": "Billy",
  "last": "Booster",
}
`;
